beavis-ci.sh

```diff
@@ -0,0 +1,256 @@
+#! /usr/bin/env bash
```

Comment:
beavis-ci.sh should not need to be checked in: it lives in its own package, and shoudl be able to be downloaded and installed if and when it's needed. Since we have travis, I'm not sure we need beavis, do we?

Reply:
Removed! And I wasn't sure how to render notebooks with Travis, but will look into it.